### PR TITLE
Lock newer node version

### PR DESCRIPTION
**Pull request by mikiquantum**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,9 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@centrifuge/gateway-server": "file:packages/server",
     "@centrifuge/gateway-lib": "file:packages/lib",
-    "@centrifuge/gateway-ui": "file:packages/ui"
+    "@centrifuge/gateway-server": "file:packages/server",
+    "@centrifuge/gateway-ui": "file:packages/ui",
+    "bignumber.js": "^9.0.0",
```
Comment (mikiquantum) [inline package.json]: Not sure if these should go in the server package instead.

Comment (ilinzweilin) [reply package.json]: I could not find where bignumber.js is used. I think bn.js is used

File: packages/server/package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "@nestjs/common": "^5.7.4",
     "@nestjs/core": "^5.7.4",
     "@nestjs/passport": "^5.1.0",
-    "bcrypt": "3.0.2",
+    "bcrypt": "3.0.8",
```
Comment (ilinzweilin) [inline packages/server/package.json:37]: 👍

File: package.json [hunk 3]
```diff
@@ -3,9 +3,11 @@
...
+    "web3-eth": "^1.2.6"
```
Comment (ilinzweilin) [inline package.json]: I think this can go in the server package.json

**Review by ilinzweilin — APPROVED** [review]
coolio. Thanks for fixing the build. approved with 2 minor comments

File: packages/server/package.json [hunk 4]
```diff
@@ -47,7 +47,9 @@
     "passport-local": "^1.0.0",
     "portable-fetch": "^3.0.0",
     "reflect-metadata": "^0.1.12",
-    "typescript": "3.2.2"
+    "typescript": "3.2.2",
+    "web3-eth": "^1.2.6",
+    "bignumber.js": "^9.0.0"
```
Comment (mikiquantum) [inline packages/server/package.json:52]: Couldn't build the package without adding it. `npm run build`

File: packages/lib/tsconfig.json [hunk 5]
```diff
@@ -17,14 +17,17 @@
...
+    "module": "esnext",
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "strictPropertyInitialization": false,
     "noImplicitAny": false,
     "jsx": "react",
     "preserveConstEnums": true,
-    "outDir": "./"
+    "outDir": "./",
+    "allowJs": true,
```
Comment (mikiquantum) [inline packages/lib/tsconfig.json]: Are any of these a problem, they were autogenerated. @ilinzweilin

**Pull request merged**